About-page/index.html

```diff
@@ -0,0 +1,192 @@
+        <br>Hosted by Arsenal Football Club</br>
+        <a href="https://www.Arsenal.com" target="_blank">www.Arsenal.com</a>
+      </h3>
+    </div>
+
+      <div class="about-logo">
+       <h2 class="mobile">
+           Football Tactics Global series <br> 2nd Edition Logo </br>
+        </h2>
+        <h2 class="desktop">
+          Football Tactics Global series 2nd Edition Logo
+       </h2>
+        <div class="line-parent">
+            <hr class="line"/>
+        </div>
+
+        <h3>
+            The Football tactics forum was created through an free online compettition in Qatar 1. Feb to 3.March
+        </h3>
+
+        <div class="logo">
+            <img src="./images/1118.jpg" alt="forum logo">
+        </div> 
+      </div>
+    <section class="past-forum-pictures">
+         <div class="previous-photos">
+            <h2 class="mobile-words">
+                See the past FTF conferences
+            </h2>
+
+            <h3 class="mobile-words2">
+                Take a look at the previous photographs from the FTC Conference that took place in England and Spain.
+            </h3>
+            <h2 class="desktop-words">
+              See the past FTF conferences
+           </h2>
+
+           <h3 class="desktop-words2">
+              Take a look at the previous photographs from the FTC Conference that took place in England and Spain.
+           </h3>
+
+            <div class="photographs photo">
+                
+                  <div class="photograph">
+                    <img src="./images/FOS-22-6.27-Turkish-Super-Lig-BeinSports.jpg" alt="conference-first-photo" class="first-image">
+                
+                  </div>
+
+                 <div class="photograph">
+                    <img src="./images/FSifztXXIAACC_2.jpg" alt="conference-second-image" class="second-image">
+                 </div>
+            </div>
+            <div class="photographs-photo-desktop">
+                
+              <div class="photograph-desktop">
+                <img src="./images/FOS-22-6.27-Turkish-Super-Lig-BeinSports.jpg" alt="conference-first-photo" class="desktop-image">
+            
+              </div>
+
+             <div class="photograph-desktop">
+                <img src="./images/FSifztXXIAACC_2.jpg" alt="conference-second-image" class="desktop-image">
+             </div>
+
+            </div>
+          </div>
+    </section>
+
+
+    <section class="sponsoring-partners">
+        <div class="sponsors">
+            <h3 class="sponsor-title">Partners</h3>
+            <div class="line-second">
+                <hr class="line2"/>
+            </div>
+        </div>
+
+        <div class="partners">
+            <div><h2 class="sponsor1">Google</h2></div>
+            <div><h2 class="sponsor2">FIFA</h2></div>
+               <div><h2 class="sponsor3">UEFA</h2></div>
+        </div>
+        <div class="english-sponsor"> 
+            <h2>English Premier League</h2>
+        </div>
+
+    </section>
```

Comment:
- [ ] Kindly you could have a div that contain all information for a particular speaker then a div that contain the image of the speaker and the background image then i second div that contain the name the `p` that must have the color `rgb(236, 82, 66)`  and the last `p` <br>
like `div class="container"> <div class="image">...</div> <div class="details">...</p></div> </div>`
Your Design
![Screenshot from 2022-07-02 01-13-29](https://user-images.githubusercontent.com/77842237/176973336-20faa621-0a40-4e86-b15f-2dd7e64fe5d5.png)

Figma design

![Screenshot from 2022-07-02 01-13-44](https://user-images.githubusercontent.com/77842237/176973361-8da81eda-f99c-4da9-97af-af494b818f77.png)